tasks/tasks.py

```diff
@@ -55,6 +61,12 @@ def _before(self):
         # Create job dir
         try:
             os.makedirs(self.data_dir)
+            if self.task_name:
+                self.test_logger = config_log_for_task(self.task_name,
```

Comment:
`self.test_logger` should be present in `__init__()`

Reply:
`self.test_logger` is not in `__init__` because the `uuid` directory is created latter. So, if something happens between the class instantiation and the `_before` method be called, the log would point to a file that doesn't exist. 

As I prefer to change the code to do not create the log with the task name (following the comment above), I'll delete the code.

Reply:
I meant it should just be *defined*, even as `self.test_logger = None`. Just to make it clear this variable exists for the object and to satisfy pylint's desires (aka `attribute-defined-outside-init`).